lib/Dialect/FIRRTL/FIRRTLTypes.cpp

```diff
@@ -1741,6 +1744,24 @@ OpenBundleType::getElementTypePreservingConst(size_t index) {
       .Default(type);
 }
 
+auto OpenBundleType::verify(function_ref<InFlightDiagnostic()> emitErrorFn,
+                            ArrayRef<BundleElement> elements, bool isConst)
+    -> LogicalResult {
```

Comment:
Whats with the trailing return type?

Reply:
Grabbed it from elsewhere in the file, I can change it :+1:.

Reply:
Gotcha, not a big deal either way :+1: